src/content/docs/en/guides/prefetch.mdx

```diff
@@ -106,17 +105,6 @@ As some navigation might not always appear as `<a />` links, you can also prefet
 </script>
 ```
 
-You can additionally configure the priority of prefetching by passing the `with` option:
-
-```js
-// Prefetch with `fetch()`, which has a higher priority.
-prefetch('/about', { with: 'fetch' });
-
-// Prefetch with `<link rel="prefetch">`, which has a lower priority
-// and manually scheduled by the browser. (default)
-prefetch('/about', { with: 'link' });
-```
```

Comment:
The `with` option is deprecated so I removed this to discourage people from using it.